kernel/base/build.gradle

```diff
@@ -24,6 +24,7 @@ repositories {
 
 compileJava {
   options.compilerArgs << '-XDignore.symbol.file'
```

Comment:
I found a way to eliminate signal warnings for only one file, I used @SuppressWarnings("sunapi").